source/less/blocks/popular-link.less

```
@@ -0,0 +1,9 @@
+.popular-link {
+  display: block;
+  font-family: "oswaldregular", "Arial", sans-serif;
```

Можно написать просто: 
```suggestion
  font-family: inherite;
```